### PR TITLE
docs: multi project

### PR DESCRIPTION
**Pull request by jon-stewart**
## Summary

Examples for multi project creation.  Referencing https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform

## How did you test this change?

terraform plan

## Issue

https://lacework.atlassian.net/browse/GROW-1502

### CODE REVIEW

**Review by dmurray-lacework — APPROVED** [review]
looks good, but is this a 'feat' or would 'docs' be a better tag?

A feat tag will do a minor version bump

File: examples/project-level-config-multi/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  description = "Map of project configuration with Lacework."
+  type        = map
+  default     = {
+    project-id-1 = "first project",
+    project-id-2 = "second project"
+  }
+}
+
+module "gcp_project_config" {
+  source = "../../"
```
Comment (afiune) [inline examples/project-level-config-multi/README.md]:
```suggestion
  source  = "lacework/config/gcp"
  version = "~> 2.0"
```

File: examples/project-level-config-multi/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  for_each   = var.projects
+  project_id = each.key
+}
+```
+
+Run Terraform:
+```
+$ terraform init
+$ GOOGLE_CREDENTIALS=account.json GOOGLE_PROJECT=my-project terraform apply
```
Comment (afiune) [inline examples/project-level-config-multi/README.md]:
```suggestion
$ GOOGLE_CREDENTIALS=account.json terraform apply
```

File: examples/project-level-config-multi/versions.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# required for Terraform 13
+terraform {
+  required_providers {
+    google = "4.36.0"
```
Comment (afiune) [inline examples/project-level-config-multi/versions.tf]:
```suggestion
    google = "~> 4.36"
```

File: examples/project-level-config-multi/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Integrate multiple Google Cloud Project with Lacework for Configuration Assessment
+The following provides an example of integrating multiple Google Cloud project with Lacework for Cloud Resource configuration assessment.
```
Comment (lacework-aaronscheiber) [inline examples/project-level-config-multi/README.md]:
```suggestion
The following provides an example of integrating multiple Google Cloud projects with Lacework for cloud resource configuration assessment.
```

File: examples/project-level-config-multi/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# Integrate multiple Google Cloud Project with Lacework for Configuration Assessment
```
Comment (lacework-aaronscheiber) [inline examples/project-level-config-multi/README.md]:
```suggestion
# Integrate Multiple Google Cloud Projects with Lacework for Configuration Assessment
```

File: examples/project-level-config-multi/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+The following provides an example of integrating multiple Google Cloud project with Lacework for Cloud Resource configuration assessment.
+
+The fields required for this example are:
+
+| Name       | Description                                                                                 | Type     |
+|------------|---------------------------------------------------------------------------------------------|----------|
+| `projects` | The id of each Project, which will be used to deploy required resources for the integration | `map` |
```
Comment (lacework-aaronscheiber) [inline examples/project-level-config-multi/README.md]:
```suggestion
| `projects` | The ID of each project, which will be used to deploy required resources for the integration | `map` |
```

File: examples/project-level-config-multi/README.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+}
+```
+
+Run Terraform:
+```
+$ terraform init
+$ GOOGLE_CREDENTIALS=account.json terraform apply
+```
+
+For detailed information on integrating Lacework with Google Cloud see [GCP Compliance and Audit Trail Integration - Configure Multiple Porjects with Terraform](https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform)
```
Comment (lacework-aaronscheiber) [inline examples/project-level-config-multi/README.md:42]:
```suggestion
For detailed information on integrating Lacework with Google Cloud, see [GCP Compliance and Audit Trail Integration - Configure Multiple Projects with Terraform](https://docs.lacework.com/onboarding/gcp-compliance-and-audit-log-integration-terraform-using-google-cloud-shell#configure-multiple-projects-with-terraform)
```

**Review by lacework-aaronscheiber — DISMISSED** [review]

**Review by afiune — APPROVED** [review]

**Pull request merged**